### ISSUE TITLE
Allow handling of non-websocket HTTP calls. 

### ISSUE THREAD

**Issue by Danielku15** [kerryjiang/SuperWebSocket#33]
It would be nice to have some sort of "Fallback-ProtocolProcessor" to handle any HTTP calls that might have come from the user. From what I could see is this would perfectly fit into the WebSocketProtocolProcessor chain.  All WebSocket protocol processors will fail on a normal HTTP call. In this case the "Fallback-ProtocolProcessor" could kick-in and handle the normal HTTP call by loading and returning some data. 

Our plan is to add a simple REST API in parallel to the WebSocket connection which allows us querying some status information of the server (e.g. connected clients, data throughput etc.). 

Sadly the protocol-processor chain is set-up hidden in the WebSocketServer.Setup method. I cannot see an easy way of adding a custom ProtocolProcessor to this chain.
